## Protein modeling

* A general introduction to structure prediction by two of the main developers of homology (Sali) and ab-initio (Baker) modeling. Baker, D; Sali, A (2001). "Protein structure prediction and structural genomics". [Science 294 (5540): 93–96](http://science.sciencemag.org/content/294/5540/93.long)

* The papers describing the algorithms implemented in Prime (Honig, Shaw, Friesner). Jacobson, MP et al. "A Hierarchical Approach to All-Atom Protein Loop Prediction," [Proteins 2004, 55, 351-367](http://onlinelibrary.wiley.com/doi/10.1002/prot.10613/abstract); 

* Comparative Modeling vs Homogloy modeling vs threading 
 
 
* seminal paper by Sander,C. and Schneider,R. (1991) [Proteins, 9, 56-68](http://onlinelibrary.wiley.com/doi/10.1002/prot.340090107/full)

* How high must the homology be for structure prediction to be reliable? Rost, B Twilight zone of protein sequence alignments, [Protein Engineering, Design and Selection, Volume 12, Issue 2, 1 February 1999, Pages 85–94](https://academic.oup.com/peds/article/12/2/85/1550637/Twilight-zone-of-protein-sequence-alignments)



### loop (de novo) modeling and side-chain prediction


* role in biologics/antibody modeling


### homology modeling and drug design

Virtual Ligand Screening Against Comparative Protein Structure Models
[Computational Drug Discovery and Design: pp 105-126](https://link.springer.com/protocol/10.1007/978-1-61779-465-0_8)

## Chemoinformatics
### chemical similarity
### QSAR
### Polypharmacology

## Molecular dynamics and free-energy methods




## Other useful links

* [Rosetta tutorials Meiler Lab (Vanderbilt)](http://www.meilerlab.org/index.php/rosetta-tutorials)
* [Rost lab (Munich)](https://www.rostlab.org/)


